.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,26 @@
+name: Update python dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "5 6 * * *"
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: "opensafely-core/setup-action@v1"
+      with:
+        python-version: "3.12"
+        install-just: true
+
+    - uses: actions/create-github-app-token@v1
+      id: generate-token
+      with:
+        app-id: 1031449  # opensafely-core Create PR app
```

Comment:
Ah. This won't work just yet, because the app is only installed on opensafely-core. I was planning to make it installable on any app and add it to all ebmdatalab and Bennett repos, now that we've verified it's all working fine on a few different opensafely-core repos

Reply:
ah, yes, somehow missed that metrics was in `ebmdatalab`; thanks!